tests/acceptance/blocks/navigationcomponent.js

```diff
@@ -0,0 +1,47 @@
+import { Selector } from 'testcafe';
+
+/**
+ * This class models user interactions with the {@link NavigationComponent}.
+ */
+export default class NavigationComponentBlock {
+  constructor () {
+    this._navList = Selector('.yxt-Nav-expanded');
+  }
+
+  /**
+   * Get an array of the labels of the nav bar tabs
+   *
+   * @returns {Array<string>}
+   */
+  async getTabLabelsLowerCase () {
+    const tabList = this._navList.child();
+    const tabListCount = await tabList.count;
+    const tabLabels = [];
+
+    for (let i = 0; i < tabListCount; i++) {
+      const nthTab = tabList.nth(i);
+      const innerText = await nthTab.find('a').innerText;
+      tabLabels[i] = innerText.toLowerCase().trim();
```

Comment:
do we have to change these to lowercase?

Reply:
I added this because locally I was getting all uppercase, but in CI, I was getting mixed case which caused the test to fail. Do you have any idea why that might be?

Reply:
Ah, I think I remember ie11 not respecting text-transform for this? Iirc can see this behavior in dev tools too if you try to console log the inner text, lowercase sgtm!